app/assets/stylesheets/modules/item-selector.scss

```diff
@@ -86,3 +86,34 @@
   }
 }
 
+.bound-withs {
+  max-height: 200px;
```

Comment:
Can we use relative units (rem or em) where appropriate?  This makes it easier to style for responsiveness to different screen sizes.